source/analytics/reports/jurisdictions/cdc-central/report-sql/SR8-State_Map_Report_of_Disease_Cases_Over_Selected_Time_Period/README.md

```diff
@@ -0,0 +1,19 @@
+# SR8: State Map Report of Disease Cases Over Selected Time Period
```

Comment:
@upasanapattnaik-eq this looks great. This can go into confluence and add only the link here. That way business can access the page.

Reply:
I have linked the Confluence page and moved the content into the Confluence page as well. Does the Github page need any context?

Reply:
@upasanapattnaik-eq add a for purpose of the report and add the link as well